tests/test_dimension.py

```diff
@@ -830,6 +831,105 @@ def test_implicit_dims(self):
 
         assert np.all(f.data == F)
 
+    def test_condition_lowering(self):
+        """
+        Test the lowering of a ConditionalDimension's condition.
+        This test makes an Operator that should indexify and lower the condition
+        passed in the Conditional Dimension
+        """
+
+        grid = Grid(shape=(3, 3))  # Define grid
+
+        # Define Function
+        g = Function(name='g', shape=grid.shape, dimensions=grid.dimensions)
+        g.data[:] = 1
+        x, y = grid.dimensions
+
+        ci = ConditionalDimension(name='ci', parent=y, condition=Le(g, 1))
+
+        f = Function(name='f', shape=grid.shape, dimensions=(x, ci))
+        Operator(Eq(f, g)).apply()
+        assert np.all(f.data == 1)
+
+    @pytest.mark.parametrize('setup_rel, rhs, c1, c2, c3, c4', [
+        # Relation, RHS, c1 to c4 used as indexes in assert
+        (Lt, 3, 2, 4, 4, -1), (Le, 2, 2, 4, 4, -1), (Ge, 3, 4, 6, 1, 4),
+        (Gt, 2, 4, 6, 1, 4)
+    ])
+    def test_relational_classes(self, setup_rel, rhs, c1, c2, c3, c4):
```

Comment:
move to `test_equation` or make a new `test_relational` that'd be consitant with splkit equation/relational

Reply:
well, the relational for now are only needed inside ConditionalDimension, and we wanna make sure that CD does lower the condition, so... I'm not sure about this one

Reply:
It is a tricky decision, although I cannot see a reason to discretize.  Both of you have a point as relationals and CDs are tightly coupled when used,

Reply:
What should I do here?

Reply:
Reminder

Reply:
leave it here it's fine